### ISSUE TITLE
Train_part

### ISSUE THREAD

**Issue by yohehe** [yohehe/pytorch_temp#2]

**Comment by yohehe** [comment]
## StandardScalerのメソッド
良く使うメソッドは次の3つ。
1. fit(X)
>配列Xの平均と分散を計算して、記憶する（変換は行わない）。
2. transform(X)
>配列Xに変換を施して、変換後の配列を返す。
3. fit_transform(X)
>配列Xに対して、fitとtransformを同時に行う。
なお、2次元配列を変換する場合、縦 (axis=0) 方向に変換が行われる。


from sklearn import preprocessing

# Fit scaler to joint train and test data
scaler = preprocessing.MinMaxScaler()
#とりあえずMinMaxで標準化する場合は以下
#データをまとめて平均と分散を計算させる。
scaler.fit(df_train.append(df_test))

# Quantile_transformの場合も同様

from sklearn.preprocessing import QuantileTransformer
trans = QuantileTransformer(n_quantiles=1000, output_distribution='normal')
trans.fit(df_train.append(df_test))

#n_quantilesをサンプル数に適用すると計算が終わらない。最適な調整を調べる必要ありそう。
